tests/test_wave_01.py

```diff
@@ -118,13 +120,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
+    assert updated_data["watched"][0].get("title") == MOVIE_TITLE_1
```

Comment:
👍🏻 Nice assert! For more complex cases (probably not needed here) if movies could share the same title, you could also add additional asserts to check the genre and rating are as expected.